src/convert.py

```diff
@@ -14,6 +24,184 @@ def save_json(fhir_resources, out_file):
     with open(out_file, 'wb') as fd:
         json.dump(fhir_resources, fd, indent=4)
 
+
+def unzip(zipped_file):
+    unzipped_file = os.path.splitext(zipped_file)[0]
+    logger.info('Unzipping %s to %s', zipped_file, unzipped_file)
+
+    with gzip.open(zipped_file, "rb") as f_in, open(unzipped_file, "wb") as f_out:
+        shutil.copyfileobj(f_in, f_out)
+
+    logger.info('Unzipping completed')
+    return unzipped_file
+
+
+def create_observation(fasta, genes, project_id, subject_id, specimen_id, specimen_name, sequence_id):
+    def create(variant_dict):
+        observation_id = str(uuid.uuid4())
+        position_value = variant_dict['@position']
+        region, position = position_value.split(':')
+        transcript = variant_dict['@transcript']
+        cds_effect = variant_dict['@cds-effect'].replace('&gt;', '>')
+        chrom, offset, ref, alt = parse_hgvs(
+            '{}:c.{}'.format(transcript, cds_effect), fasta, genes)
+
+        observation = {
+            'resourceType': 'Observation',
+            'identifier': [{
+                'system': 'https://lifeomic.com/observation/genetic',
+                'value': '{}:{}:{}:{}'.format(chrom, offset, ref, alt)
```

Comment:
@dmmiller612 do you think I should hash this?

Reply:
Good question, probably no? We could rely on the ui to do that.

Reply:
yeah, I didn't know if hashing now would make it easier when consuming these on the analytics side to merge into the master variant table.

Reply:
It would, but we can leverage the ui there.